compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -750,6 +750,7 @@ void buildLLVMCPUCodegenConfigurationPassPipelineImpl(
       // way to late and should insted be be done during lowering to LLVM.
       .addPass(createExpandF16OpToF32Pass)
       .addPass([&]() { return createCPUMaterializeEncodingPass(); })
+      .addPass(createMaterializeEncodingIntoNopPass)
```

Comment:
I think we don't need it. It is already handled by `CPUMaterializeEncodingPass`.

Reply:
oh right, we reverted the changes in `MaterializeHomogeneousEncodings` so this is outdated code.